README.md

```diff
@@ -59,7 +59,7 @@ The available options are:
 
 Name                   | Description
 ---------------------- | ----------------------------------------------------------
-`requireBlueprintName` | Require that the API Blueprint have a title (default: false)
+`requireBlueprintName` | Require parsed API Blueprints have a title (default: false)
```

Comment:
I'd say

API Blueprint is a format

API blueprint is a file in API Blueprint format

So: 

"Require parsed blueprints have a title"